docs/source/configuration.rst

```diff
@@ -75,25 +125,14 @@ Specify Configuration
 YAML files
 ~~~~~~~~~~
 
-You can specify configuration values in YAML files like the following:
-
-.. code-block:: yaml
-
-   array:
-      chunk-size: 128 MiB
-
-   distributed:
-     logging:
-       distributed: info
-       bokeh: critical
-       tornado: critical
+You can specify configuration values in YAML files. The default YAML files for ``dask`` 
+and ``distributed`` can be found in:
 
-     scheduler:
-       work-stealing: True
-       allowed-failures: 5
+``dask``: `dask.yaml
+<https://github.com/dask/dask/blob/main/dask/dask.yaml>`_,
 
-       admin:
-         log-format: '%(name)s - %(levelname)s - %(message)s'
+``distributed``: `distributed.yaml
+<https://github.com/dask/distributed/blob/main/distributed/distributed.yaml>`_,
```

Comment:
In this case, I think displaying the contents of an example YAML file is useful as this section isn't talking specifically about the default configuration values, but rather how users can define their own custom YAML config files.

Reply:
Is it possible to specify only one part of the config file and the rest would default to the default? If so I wonder what's the most common edit for the YAML file, Is it the workers memory? i.e this portion of the config
```
'memory': {'pause': 0.8,
           'rebalance': {'measure': 'optimistic',
                         'recipient-max': 0.6,
                         'sender-min': 0.3,
                         'sender-recipient-gap': 0.1},
           'recent-to-old-time': '30s',
           'spill': 0.7,
           'target': 0.6,
           'terminate': 0.95},
```

In that case how would the YAML that modifies this portion look like? (I haven't experimented with this so I'm not sure how exactly works.)

Reply:
I added a `YAML` example, but I'm not sure if it's correct. @jrbourbeau  let me know if we need to fix it or add something else.